test/SpokePool.Relay.ts

```diff
@@ -682,6 +682,7 @@ async function testfillRelayWithUpdatedDeposit(depositorAddress: string) {
         updatedMessage,
         consts.modifiedRelayerFeePct, // Applied relayer fee % should be diff from original fee %.
         false,
+        "0",
```

Comment:
How did this ever work? Do these tests pass?

Reply:
Yep! These tests pass - below are the outputs of the calls to this function commented above.

```
    ✔ Can fill relay with updated fee by including proof of depositor's agreement (38ms)
    ✔ EIP1271 - Can fill relay with updated fee by including proof of depositor's agreement
```